test/testfiles_for_unittests/compressed.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
```

Comment:
Add a comment here saying how you produce the .o files from this file (exact gcc/compiler command line)